api-client/src/endpoints/hooks/useLoadFromEndpoint.ts

```diff
@@ -1,24 +1,24 @@
-import determineApiUrl from '../determineApiUrl'
 import { Payload } from 'api-client'
 import { useCallback, useEffect, useState } from 'react'
-type RequestType<T> = (apiUrl: string) => Promise<Payload<T>>
+
+type RequestType<T> = () => Promise<Payload<T>>
+
 export const loadFromEndpoint = async <T>(
   request: RequestType<T>,
-  setData: (arg0: T | null | undefined) => void,
-  setError: (arg0: Error | null | undefined) => void,
-  setLoading: (arg0: boolean) => void
-) => {
+  setData: (data: T | null) => void,
+  setError: (error: Error | null) => void,
+  setLoading: (loading: boolean) => void
+): Promise<void> => {
   setLoading(true)
 
   try {
-    const apiUrl = await determineApiUrl()
```

Comment:
We used determineApiUrl for choosing test-cms or cms. Is this no longer needed? Or where gets it checked now?

Reply:
Oke I see it is in the Containers now. I think it would be a lot nicer if it would be central and only in one place. A container should not be in charge for determining the correct cms-url, should it? Is there a better place to put it?

Reply:
I moved `useLoadFromEndpoint` to the api-client. Therefore it is not possible to keep this functionality in the `useLoadFromEndpoint` hook. In theory it would be possible to create another wrapper around it in the native project though. I would not do this now already though.